include/libtorrent/create_torrent.hpp

```diff
@@ -73,9 +75,37 @@ see LICENSE file.
 //
 namespace libtorrent {
 
+TORRENT_VERSION_NAMESPACE_4
+
 	// hidden
 	using create_flags_t = flags::bitfield_flag<std::uint32_t, struct create_flags_tag>;
 
+	// This represents one file in the torrent file to be created
+	struct TORRENT_EXPORT create_file_entry
```

Comment:
Always felt this name confusing since it isn't any kind of "callable".

Reply:
the underlying (msvc) feature `TORRENT_EXPORT` resolves to is `__declspec(dllexport)`, which means this type will be exported by the shared library. For types, in addition to exporting member functions, that means exporting the vtable and RTTI. (RTTI is used for `typeinfo` and `dynamic_cast`)

Reply:
My comment refers to class names. And yes, I missed a bit (due to the inconvenience of navigating GitHub on a smartphone), so it had to be under the `create_torrent` name.

Reply:
I would expect it to be something like `torrent_creator` since it isn't a "callable".

Reply:
ah, yes. That would be a better name probably

Reply:
So can rename it (and others like it)? This could be done in the `master`, and in the current branches you could create aliases, like you have already done `file_layout` for `file_storage`.